.github/workflows/main.yml

```
@@ -0,0 +1,67 @@
+name: Build and Publish
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - '*'
+
+jobs:
+  build_template:
+    runs-on: ubuntu-latest
+    
+    env:
+      TOOLCHAIN_UPDATE: 10.3-2021.10
```

```suggestion
      TOOLCHAIN_UPDATE: 13.3.rel1
```
PROS uses v13.3rel1 since https://github.com/purduesigbots/pros/pull/703.